lib/htmlValidator.js

```diff
@@ -48,29 +51,38 @@ module.exports = function(app, callback) {
 
         validatorProcess.stdout.on('data', (data) => {
           if (`${data}`.includes('Initialization complete')) {
+            validatorDisabled = false;
+            clearTimeout(validatorTimeout);
             resolve();
           }
         });
 
         validatorProcess.stderr.on('data', (data) => {
           if (`${data}`.includes('No Java runtime present, requesting install.')) {
-            console.error('error: No Java runtime detected. Check to make sure Java is installed and in your path.');
+            console.error(('There was an error in authentication. Please make sure to have Java installed.').red);
+            validatorDisabled = true;
             reject();
           }
-          else {
```

Comment:
Don't mean to step on your toes @Autre31415, I have just been helping @JoshL18 in person with this. You have to be careful with the removal of this else, (is there a case when we will get something in standard error and the validator can't start? If so we will want this else.

I may have told you to remove this @JoshL18, if I did sorry about that.